packages/api/src/apps/api/app.ts

```diff
@@ -89,10 +90,14 @@ async function run(env: ProcessEnv) {
   console.log("Updated emp state");
   await services.erc20s.update();
   console.log("Updated tokens");
-  await services.collateralPrices.update();
-  console.log("Updated Collateral Prices");
   await services.syntheticPrices.update();
   console.log("Updated Synthetic Prices");
+
+  // backfill price histories
+  await services.collateralPrices.backfill(moment().subtract(1, "month").valueOf());
```

Comment:
Is this just a temporary measure until we have a database up and running?  And are we planning to do the same for EMP data? Or do we only need price history?

Reply:
currently we only need price data for charts for tvl.  Theres no feature as far as i know needed for emp state. Once we have database this might still be useful, but will probably turn it into an environment variable and disable it if not explicitly needed.